### ISSUE TITLE
Clear Button for JSON input

### ISSUE THREAD

**Issue by p-shah** [zgrossbart/jdd#2]
Awesome service but there's no button to clear the input.

**Comment by zgrossbart** [comment]
Thank you for the comment.  You can press the "Perform a new diff" button, remove the input manually, or refresh the page.  What kind of button are you looking for?

**Comment by p-shah** [comment]
Just to have the option to not have to manually each time. But if it was designed purposely that way, my bad.

**Comment by zgrossbart** [comment]
No need for a mea culpa.  I'm really curious what you mean.  Were you looking for a clear button while you were using the page?  When did you want it?  I want to make this as usable as possible.  Thank you for the help.

**Comment by p-shah** [comment]
Yeah I was just using JSON diff service and noticed everytime I tried to perform a new diff I had to manually clear the input on each side. I figured to a button clear above each of the inputs would be useful. Thanks for the service.. Super helpful.

**Comment by zgrossbart** [comment]
That makes sense.  I was torn about that feature.  Right now I don't clear the inputs when you perform a new diff based on the idea that you might want to change the existing input rather than have something totally new.  What do you think?  Does it make sense to clear the input there?

I'm also trying to keep everything really simple and not add more buttons that an absolutely have to.

**Comment by saurabheights** [comment]
@zgrossbart : My two cents here: Although, I am wary of this functionality as well. However, a warning regarding "Do you want to really clear all data?" should be sufficient. Clearing the input manually is more effort than the two clicks, one for the clear button and one for the confirm.
